src/modules/hyprland/workspaces.cpp

```diff
@@ -202,6 +316,7 @@ void add_or_remove_class(const Glib::RefPtr<Gtk::StyleContext> &context, bool co
 void Workspace::update(const std::string &format, const std::string &icon) {
   auto style_context = button_.get_style_context();
   add_or_remove_class(style_context, active(), "active");
+  add_or_remove_class(style_context, is_empty(), "persistent");
 
   label_.set_markup(fmt::format(fmt::runtime(format), fmt::arg("id", id()),
```

Comment:
persistent workspaces makes implementing label id impossible, so this should be removed and doc updated

Reply:
Do you mean just remove the `fmt::arg("id", id())` from this line?

Reply:
yes